### ISSUE TITLE
Unused variable issue when variable have same name as package

### ISSUE THREAD

**Issue by ronindev** [go-lang-plugin-org/go-lang-idea-plugin#1934]
Same as issue https://github.com/go-lang-plugin-org/go-lang-idea-plugin/issues/1916
Sample code: 
```go
package main

import (
	"fmt"
	"os/user"
)

func main() {
	fmt.Println(getUserName())
}

func getUserName() string {
	user, err := user.Current()
	if err != nil {
		panic(err)
	}
	return user.Username
}
```

![1999](https://cloud.githubusercontent.com/assets/921324/10600697/2a1faf5c-7715-11e5-8833-f5b8f6240824.png)

IntelliJ IDEA 14.1.5
Build #IC-141.2735, built on September 22, 2015
JRE: 1.8.0_51-b16 x86
JVM: Java HotSpot(TM) Server VM by Oracle Corporation

Go plugin version 0.9.560

Windows 8 x64

**Comment by ronindev** [comment]
Issue occure with sample code from #1916 if you using methods or fields of variable like this:

```go
func TestVariable(t *testing.T) {
	time := time.Now()
	t.Logf("%s", time.String()) // <-- here
}
```

**Comment by dlsniper** [comment]
Thanks for the detailed report, I can confirm this happens as well.

**Comment by ignatov** [comment]
Fixed in 0.9.561
